docs/installation.rst

```diff
@@ -19,6 +19,27 @@ You can also install it into a new virtual environment:
    $ source ~/.virtualenvs/cernopendata-client/bin/activate
    $ pip install cernopendata-client
 
+In order to use Pycurl / xrootd related functionality, additional
+dependencies are required.
+
+The ``pycurl`` "extra" install flavor
```

Comment:
Instead of introducing new sections (1.2 and 1.3), this should be living as part of the 1.1 section.

Hence you can change formatting to something like:

This will use Python **requests** library for data download.

If you would like to use a more performing **pycurl** library for downloading data, please add the `pycurl` flavour while installing `cernopendata-client`:

```
$ pip install cernopendata-client[pycurl]
```

Note that you need to have curl (..link...) and openssl (...link...) set up on your system.

Additionally, if you want to use an even more powerful **XRootD** protocol for downloading data, please add the `xrootd` flavour when installing `cernopendata-client`:

```
$ pip install cernopendata-client[xrootd]
```

Note that you need to have XRootD (...link...) set up on your system.

Finally, note that you can combine both flavours:

```
$ pip install cernopendata-client[pycurl,xrootd]
```

if you wish to have both capabilities.

Reply:
Done 👍